### ISSUE TITLE
jQuery Mobile dead?

### ISSUE THREAD

**Issue by wallym** [jquery/jquery-mobile#8546]
I am guessing that jQuery Mobile is dead.  I don't see any updates on it from the meeting notes standpoint.  It has been 6 months since the meeting notes were updated.  The last major update was 3 years ago.  I've got a web product that uses it in certain situations.  It gets used for some form element styling and that's about it.  Every year at this time, I look to make sure that all of my frameworks are up to date, and jQM appears to be dead.  I didn't use it much when we started because I didn't want to be tied to heavily to it.  What is the recommended simple framework to replace it?  I don't want to ride too long on a dead framework.

Wally

PS.  I'm not trying to be rude.  I'm just looking at facts.

**Comment by xilaworp** [comment]
A status update would indeed be most welcome, also it would  be interesting to know how JQuery Mobile's status as a JS Foundation project will affect future development.

**Comment by JimHapenney** [comment]
Yep, getting bit worried myself.  Some update about 1.5 from someone would be much appreciated.  Been using jqm since the beginning, don't really wanna learn/port to a new framework, happy with jqm, bugs and all, but not many since 1.4.5.  A few, but workarounds are easy.

**Comment by ElliotNB** [comment]
I'm concerned about this myself. It looks like the jQuery Mobile team last had a meeting on September 29th, 2016: https://docs.google.com/spreadsheets/d/11-mngR57h54cIOjGDE2Z6LjaiyRgDwlcvKUzZn2Fw6M/edit#gid=2121311965

There isn't anything in the meeting notes suggesting that jQuery Mobile is being abandoned, but there certainly hasn't been any recent activity in the repo.

**Comment by primet21** [comment]
I second this. Would be nice to get info on JQM's status.

**Comment by arschmitz** [comment]
Sorry i did not see this soon i was un available for the holidays. jQuery Mobile is not dead at all. There have been many ups and downs in the team recently that have lead to unexpected delays in the release of 1.5 however the alpha is complete and has been tagged earlier this week

There will be an official announcement tomorrow again i apologize in the delay in responding to this issue

**Comment by vincedan** [comment]
good news...  "write less, do more" , we need this framework alive.

**Comment by arschmitz** [comment]
Just small update sorry we didn't get the official announcement out we had a minor infrastructure issue and didn't want to announce until it was fixed the release is available on the cdn and the new demos are available until then though. 

http://code.jquery.com/mobile/
http://demos.jquerymobile.com/1.5.0-alpha.1/

**Comment by JimHapenney** [comment]
The demos look great arschmitz!

One concern though, appears you're loading jQuery 1.11 for the demos.  Will jQM 1.5 be jQuery 3.1 compatible?

**Comment by arschmitz** [comment]
@JimHapenney Yes it is fully compatible with jQuery 3.x

**Comment by ElliotNB** [comment]
I'm very glad to see these updates! The jQuery Mobile framework is a very important part of our development stack. While other mobile frameworks often impose their own strict rules and convention, JQM is wonderfully flexible and versatile. +1 to @vincedan "we need this framework alive".

**Comment by CharLiPPS** [comment]
Excellent News Alex. That's great news!!!

**Comment by deblocker** [comment]
Backward-compatibility & bugfixes are more worth than a new release every three months.

**Comment by arschmitz** [comment]
@deblocker we fixed several hundred issues during the 1.5 development cycle reducing the issue count to its lowest point since before 1.0

**Comment by cyberbeat** [comment]
I am very happy, that you are working on version 1.5!

But I tried to replace 1.4.5 in my project with master today, and reverted after a while. Too many things broke. I hope backwards compatibility will worked on some more? For example the icon-class names and data-attributes, the header-data-attributes.

And I found a bug in the demos, which appeared in my project too: http://view.jquerymobile.com/master/demos/navbar/ in the section "Navbars in headers". The button in the toolbar is not rendered correctly.

**Comment by fgarzonhz** [comment]
Please can you post in your blog what are the dates estimate to delivery maybe a RC and RTM , so we can help start test with our project and report any thing we found.
Thanks for your work and continue as this. Congratulations for the effort you is doing.

**Comment by ElliotNB** [comment]
I second what @fgarzonhz has said. The jQuery Mobile project needs some kind of blog update or announcement . Otherwise people will continue to assume that jQuery Mobile is dead or very close to it.

We love jQuery Mobile and want to see it continue to thrive! Thank you all for your efforts! @arschmitz @jaspermdegroot

**Comment by cgack** [comment]
Hey folks. I'm not @arschmitz or @jaspermdegroot (obviously) but I'm a member of the team.  We have done a ton of work over the last few years for 1.5. Lately with everyone's day jobs it has been hard to even coordinate a weekly meeting to update everyone on the status of the project so we know how to move forward with the release. This happens with a volunteer workforce that is distributed around the planet. Hopefully we can have more info and a blog post soon.

**Comment by vankampenp** [comment]
If the release is nearly there, that is good new, but someone as jQuery needs to start to do some communication fast to see if this patient can be revived.

**Comment by arschmitz** [comment]
Sorry we are working through some technical issues cause our website not to update as soon as its resolved you will see updates on the various JQM websites

**Comment by SuhniCalifornia** [comment]
sorry, but this looks like a mess. i appreciate the intricacies of development - but this really is embarrassing to witness

**Comment by vankampenp** [comment]
arschmitz I am looking forward to the announcement

**Comment by arschmitz** [comment]
We got the main website fixed this afternoon and you can now see the live changelog http://jquerymobile.com/changelog/1.5.0-alpha1/ the api is next once that is done the blog will follow

**Comment by vankampenp** [comment]
A long list, should it not also say jQuery 3.X compatibility?

**Comment by arschmitz** [comment]
@vankampenp Yes we will add that the changelog is generated from commits and there was no specific commit for that so it was missed thank you

**Comment by DropkickSteve** [comment]
@SuhniCalifornia  jq mobile is an open source framework, developed by people who have other professional business to do. If you are ungrateful or dissatisfied you can choose to stay away from the community

**Comment by windsurfmarkt** [comment]
We still working on a new version for our page with jqm an we hope for the next version

**Comment by ElliotNB** [comment]
How are things coming along @arschmitz ?

**Comment by SuhniCalifornia** [comment]
@DropkickSteve 
re:  jq mobile is an open source framework, developed by people who have other professional business to do. If you are ungrateful or dissatisfied you can choose to stay away from the community

i stand by my comment & honest expression regarding the progress of this project
best of luck & maybe another team will come to the rescue
sometimes it is wise to admit when you are in over your head
time has taught me that much
so with much sadness
consider me gone

**Comment by arschmitz** [comment]
We hope to have the api fixed and up this week ( we thought last week but there was some debate on best way to document the version of UI we are using ) as soon as that is live we will publish the blog post. 

After that we will give it a few week for people to test and report issues before releasing the beta and hopefully final soon there after anyone wanting can use it now though by checking out the 1.5-alpha.1 tag and building

**Comment by mindchasers** [comment]
Let's do it Alexander.  We're ready to help.

**Comment by frankie-loves-jesus** [comment]
Really looking forward to the update!

Meanwhile I found http://developer.telerik.com/featured/is-jquery-still-relevant/ to provide a much needed perspective on this whole issue.

**Comment by robcaldecott** [comment]
We have been using jQM for our mobile web apps since 2011. We switched to using React 18 months ago and put together some React components to wrap the jQM CSS and this kept us going for a while longer. But the lack of project updates and a UI that is looking increasingly stale had pushed us to a different framework (material-ui). But I would like to express my gratitude for all the efforts that have gone into this product over the years: but it feels like the world has moved on (and increasingly away from jQuery). I have been a huge advocate of jQM over the years but it's time to bid farewell.

It was a great run though and I still come across sites that use it. Thank you again for letting us stand on the shoulders of giants and provide our customers with a great mobile web app experience for over half a decade. I wish you every success.

**Comment by vankampenp** [comment]
Well I have an existing site, and no desire to change JQM, as long as it keeps compatible with different browsers and devices.
So I am looking forward to the new release, get rid of the old code that supports old browsers. 

Glad to help by testing it once it is stable.

**Comment by apsdehal** [comment]
Hi everyone,

API documentation for 1.5 is up at http://api.jquerymobile.com/. You can download and start using/testing it from http://jquerymobile.com/download-builder/ using 1.5 or latest option there. We will soon publish a blog post announcing its official release.

**Comment by arschmitz** [comment]
Everything for 1.5 is up and running now and the blog has been published so i'm going to close this.

I would just like to mention to everyone that expressed support in this issue that we are looking for more help and new team members going forward to keep progress moving at a much faster pace. If any one is interested in helping with development of jQuery mobile feel free to join our weekly meetings on irc or reach out to me directly

**Comment by vincedan** [comment]
thanks, @arschmitz and other guys who contributed and is contributing on juerymobile.  thanks you for your effort.

**Comment by sixtyfive** [comment]
@arschmitz, I just tried getting JQM 1.5 (built from git) to work with a current jquery-git, but can't get past `TypeError: $.event.props is undefined`, the same as when trying with JQM 1.4. Am I hitting a new incompatibility? Any chance to make it work?

**Comment by arschmitz** [comment]
@sixtyfive Can you please open a new issue for this its not related to this issue
